common/djangoapps/util/tests/test_db.py

```diff
@@ -215,11 +216,14 @@ class MigrationTests(TestCase):
     """
     Tests for migrations.
     """
-    @skipIf(isinstance(settings.MIGRATION_MODULES, NoOpMigrationModules), 'Skip in case of NoOpMigrationModules')
+    # @skipIf(isinstance(settings.MIGRATION_MODULES, NoOpMigrationModules), 'Skip in case of NoOpMigrationModules')
     def test_migrations_are_in_sync(self):
         """
         Tests that the migration files are in sync with the models.
         If this fails, you needs to run the Django command makemigrations.
         """
-        with self.assertRaises(SystemExit):
-            call_command('makemigrations', '-e')
+
+        # Ensure that migrations are enabled before calling the migrations task
+        with patch.dict(settings.MIGRATION_MODULES, {}, clear=True):
```

Comment:
Och you're right. Funny after suggesting that to others that I would forget to do it myself. I'll do that next.